NEWS.md

```diff
@@ -99,12 +99,17 @@ based on [Common Terminology Criteria for Adverse Events (CTCAE) v5.0](https://c
 
 ## Documentation
 
+- New vignette "Creating a PK NCA ADaM (ADPC/ADNCA)" (#1639)
+
```

Comment:
Is this intended? Looks like a duplicate. And somehow the sections were mixed up.